pkg/roachprod/cloud/gc_aws.go

```diff
@@ -219,9 +228,9 @@ func deleteKeyPair(EC2Client *ec2.Client, keyPairName string) error {
 	return nil
 }
 
-// GCAWSKeyPairs tags keypairs created by roachprod with IAMUserName and CreatedAt if untagged and
+// gCAWSKeyPairs tags keypairs created by roachprod with IAMUserName and CreatedAt if untagged and
 // deletes keypairs created by previous users/employees (TeamCity keypairs are deleted after 10 days).
-func GCAWSKeyPairs(l *logger.Logger, dryrun bool) error {
+func gCAWSKeyPairs(l *logger.Logger, dryrun bool) error {
```

Comment:
Nit: awkward capitalization; suggest renaming to `gcAWSKeyPairs`.

Reply:
Done